File: ReactJS-WebApp Gym/src/models/Enquiry.js
//data model of a Enquirey

export default class Enquiry {
    constructor(name, email, mobile, query, status, cseRemarks) {
        this.name = name;
        this.email = email;
        this.mobile = mobile;
        this.query = query;
        this.status = status;
        this.cseRemarks = cseRemarks;
    }

}

File: .Net Core - GoldiesGym API/EnquiryService/Models/Enquiry.cs
using MongoDB.Bson.Serialization.Attributes;

namespace EnquiryService.Models
{
    /*
     * Model class definition for Enquiry entity
     */
    [BsonIgnoreExtraElements]
    public class Enquiry
    {
        [BsonId]
        public int EnquiryId { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Mobile { get; set; }
        public string Query { get; set; }
        public string Status { get; set; }
        public string CseRemarks { get; set; }
    }
}